holdings_maintenance/pds4/pds4indexshelf.py

```
@@ -0,0 +1,533 @@
+                                  filename_keylen=pdsf.filename_keylen)
+
+        table.index_rows_by_filename_key()      # fills in table.filename_keys
+        childnames = table.filename_keys
+        index_dict = {c:table.row_indices_by_filename_key(c)
+                      for c in childnames}
+
+        logger.info('Rows tabulated', str(len(index_dict)), force=True)
+
+        latest_mtime = max(os.path.getmtime(pdsf.abspath),
+                           os.path.getmtime(pdsf.label_abspath))
+        dt = datetime.datetime.fromtimestamp(latest_mtime)
```

Ruff (0.14.0)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)